apps/launch.html.markerb

```diff
@@ -7,73 +7,86 @@ order: 10
 redirect_from: /launch/
 ---
 
-With Fly Launch, you can create a brand new app on Fly.io by running this command from the source directory of your project:
+Fly Launch helps you configure and orchestrate your app. 
+
+To create a brand new app on Fly.io, run this command from the source directory of your project:
 
 ```cmd
 fly launch
 ``` 
 
-If you're in a hurry to try your first launch, we also have a condensed [Hands-on](/docs/hands-on/) that launches a super-simple demo app. You might also be interested in our [language- and framework-specific launch guides](/docs/languages-and-frameworks/).
+If you're in a hurry to try your first launch, we have a condensed [Hands-on](/docs/hands-on/) that launches a super-simple demo app. Or check out our [language- and framework-specific launch guides](/docs/languages-and-frameworks/).
 
 ## Ingredients for a successful `fly launch`
 
 Here are the components of a successful launch, ready for the first deployment:
 
-* a way to get a Docker image that Fly.io can use to boot up a Machine with your app running
-* Fly.io-specific [configuration](/docs/reference/configuration/) for the app, in the form of a `fly.toml` file
-* optional platform resources such as [public IP addresses](/docs/reference/services/), [Fly Volumes](/docs/reference/volumes), [app secrets](/docs/reference/secrets/), [Fly Postgres](/docs/postgres/) clusters, integrated resources like [Upstash for Redis](/docs/reference/redis/)
+* A Docker image that Fly Launch can use to boot up a Machine for your app to run on.
+* A `fly.toml` file for your app's Fly.io-specific [configuration](/docs/reference/configuration/).
+* Optional platform resources such as [public IP addresses](/docs/reference/services/), [Fly Volumes](/docs/reference/volumes), [app secrets](/docs/reference/secrets/), [Fly Postgres](/docs/postgres/) clusters, and integrated resources like [Upstash for Redis](/docs/reference/redis/).
 
 ## Framework launch scanners
 
-Depending on your project, `fly launch` may be able to look at your app's source code and get through that ingredient list, straight to a ready-to-deploy Fly App. This is most likely for frameworks on which Fly.io has people specializing full time. Right now that's [Elixir/Phoenix](/docs/elixir/), [Laravel](/docs/laravel/), [Rails](/docs/rails/), and [Django](/docs/django/), but we also have launch guides for other [languages and frameworks](/docs/languages-and-frameworks/).
+Depending on your project, `fly launch` may be able to look at your app's source code and go straight to a ready-to-deploy Fly App. This is most likely to work for frameworks on which Fly.io has people specializing full time. Right now that's [Elixir/Phoenix](/docs/elixir/), [Laravel](/docs/laravel/), [Rails](/docs/rails/), and [Django](/docs/django/), but we also have launch guides for other [languages and frameworks](/docs/languages-and-frameworks/).
```

Comment:
Is it worth finding a clear way to refer back to that list of "components of a successful launch", to make it clear what `fly launch` can do, and that it doesn't always manage that?